pages/search/views/index.jsx

```diff
@@ -12,106 +10,29 @@ import {
-import projectFormatters from '@asl/pages/pages/project/formatters';
+import formatters from '../formatters';
 
 function uppercaseFirst(str) {
   return `${str[0].toUpperCase()}${str.substring(1)}`;
 }
 
-const formatters = {
-  establishments: {
-    name: {
-      format: (name, establishment) => {
-        return <Link page="establishment.dashboard" establishmentId={establishment.id} label={name} />;
-      }
-    },
-    inspector: {
-      format: (inspector, establishment) => {
-        return establishment.asru.filter(p => p.asruInspector).map(profile => (
-          <p key={profile.id}>{`${profile.firstName} ${profile.lastName}`}</p>
-        ));
-      }
-    },
-    spoc: {
-      format: (spoc, establishment) => {
-        return establishment.asru.filter(p => p.asruLicensing).map(profile => (
-          <p key={profile.id}>{`${profile.firstName} ${profile.lastName}`}</p>
-        ));
-      }
-    }
-  },
-  profiles: {
-    lastName: {
-      format: (lastName, profile) => {
-        return <Link page="globalProfile" profileId={profile.id} label={`${profile.firstName} ${lastName}`} />;
-      }
-    },
-    establishments: {
-      format: (establishments, profile) => {
-        if (profile.asruUser) {
-          return 'ASRU';
-        }
-        return sortBy(establishments, 'name').map(establishment => (
-          <p key={establishment.id}>
-            <Link page="establishment.dashboard" establishmentId={establishment.id} label={establishment.name} />
-          </p>
-        ));
-      }
-    }
-  },
-  projects: {
-    title: {
-      format: (title, project) => {
-        return (
-          <Fragment>
-            <Link page="project.read" establishmentId={project.establishment.id} projectId={project.id} label={projectTitle(project)} />
-            { project.isLegacyStub ? ' - Partial record' : '' }
-          </Fragment>
-        );
-      }
-    },
-    establishment: {
-      format: establishment => {
-        return <Link page="establishment.dashboard" establishmentId={establishment.id} label={establishment.name} />;
-      }
-    },
-    status: projectFormatters().status,
-    licenceHolder: {
-      format: ({ id, firstName, lastName }) => {
-        return <Link page="globalProfile" profileId={id} label={`${firstName} ${lastName}`} />;
-      }
-    },
-    expiryDate: {
-      format: (date, model) => {
-        if (!date || ['revoked', 'transferred'].includes(model.status)) {
-          return '-';
-        }
-        return <ExpiryDate
-          date={date}
-          dateFormat="D MMM YYYY"
-          showNotice={model.status === 'active' ? 11 : false}
-        />;
-      }
-    }
-  },
-  'projects-content': {
-    title: {
-      format: (title, project) => <ProjectSearchResult project={project} />
-    }
-  }
-};
+export default function Index () {
+  const { profile, searchType } = useSelector(state => state.static);
+  const { filters, pagination } = useSelector(state => state.datatable);
```

Comment:
Joe will be delighted.

Reply:
😄  I've always hated this noise:
```
const mapStateToProps = ({	
  static: { profile, searchType },	
  datatable: { filters: { options, active } }	
}) => ({ profile, searchType, hasFilters: !!options.length, searchTerm: active });
```

Hooks are a billion times nicer.